packages/core/docs/integrations/adyen.md

```diff
@@ -143,6 +145,34 @@ const afterPayAndOrder = async ({ order }) => {
 ### Paypal configuration
 Configuration of PayPal is well-described in [Adyen's documentation](https://docs.adyen.com/payment-methods/paypal/web-drop-in).
 
+### Klarna configuration
+To enable Klarna, you have to add a new payment method in Adyen's dashboard. Then you should add specified methods to the `availablePaymentMethods` array in `nuxt.config.js`:
+```js
+// nuxt.config.js
+
+export default {
+  modules: [
+    ['@vsf-enterprise/adyen/nuxt', {
+      availablePaymentMethods: [
+        'scheme',
+        'paypal',
+        'klarna',
+        'klarna_account',
+        'klarna_paynow'
+      ],
+      // ...
+    }]
+  ]
+};
+```
+
+Read [Adyen's document aboout the Klarna](https://docs.adyen.com/payment-methods/klarna#supported-countries) to check which Klarna payment method is available for which country.
+
+:::warning Phone number
+If your users may provide phone number then make sure it is with **area code**. Otherwise, Klarna will throw an error because of unproper phone number.
```

Comment:
```suggestion
If your users can provide a phone number then make sure it is with **area code**. Otherwise, Klarna will throw an error because of an improper phone number format.
```